### PR TITLE
Fixed speed dial when contact has multiple phone numbers

### PR DESCRIPTION
**Pull request by Merkost**
https://github.com/SimpleMobileTools/Simple-Dialer/assets/56008710/5cbc2db5-16e5-4f30-8f68-7430e32fa575

### CODE REVIEW

**Comment by tibbi** [comment]
and avoid showing the number in Speed Dial screen if the contact has 1 number only

**Comment by Merkost** [comment]
> and avoid showing the number in Speed Dial screen

How will user know about what number is picked then?

File: app/src/main/kotlin/com/simplemobiletools/dialer/adapters/PhonesAdapter.kt [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.simplemobiletools.commons.models.PhoneNumber
+import com.simplemobiletools.dialer.R
+import com.simplemobiletools.dialer.activities.SimpleActivity
+import kotlinx.android.synthetic.main.item_phone_number.view.select_phone_number_holder
+import kotlinx.android.synthetic.main.item_phone_number.view.select_phone_number_radio
+
+class PhonesAdapter(
```
Comment (tibbi) [inline app/src/main/kotlin/com/simplemobiletools/dialer/adapters/PhonesAdapter.kt:12]: guess this file isnt needed at all anymore, if we use the casual radiogroup dialog

**Comment by tibbi** [comment]
should be good now, thanks

**Pull request merged**